desktop-environment.el

```diff
@@ -134,6 +134,14 @@ replaced by the desired new volume level."
   "Shell command toggling between muted and working."
   :type 'string)
 
+(defcustom desktop-environment-volume-get-state-regexp "\\[on]\\|\\[off]"
+  "Regular expression matching volume state.
+
+This regular expression will be tested against the result of
+`desktop-environment-volume-get-state-command' and group 1 must
```

Comment:
this command doesn't exist

Reply:
Oh sorry, at first I was going for a different thing, forgot to change this line.

Reply:
Oh sorry, at first I was trying another solution but I forgot to remove this line.